recipes/_update_packages.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Cookbook Name:: cfncluster
+# Recipe:: _update_packages
+#
+# Copyright 2013-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+case node['platform_family']
+when 'rhel'
+  execute 'yum-update' do
+    command "yum -y update"
```

Comment:
We've accrued a bunch of old kernels in the CentOS AMIs as a result of all the upgrades along the way.  We should add a `package-cleanup --oldkernels --count=1` here to remove the older kernels.

Reply:
I can add this line in recipe. You have the same in the new packer script. Adding in recipe doesn't harm us, let me know if this comment still holds as of now.

Reply:
it's actually `package-cleanup -y --oldkernels --count=1`, and yeah, we should run it more regularly.